juicy-checkbox-group-item.html

```diff
@@ -161,7 +161,7 @@
             </template>
             <template is="dom-if" if="{{!checkboxHidden}}">
                 <div id="checkboxItemContainer">
-                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox">
+                    <div id="checkboxItem" data-checkable$="{{checkable}}" data-checked$="{{checked}}" on-click="toggleCheckbox" on-keypress="keypressCheckbox" tabindex="0">
```

Comment:
Difference between KichetnSink and JuicyCheckboxItem is that in KitchenSink we have `input`, in juicy we have `div`